components/cluster/command/scale_in.go

```diff
@@ -63,7 +63,7 @@ func newScaleInCmd() *cobra.Command {
 				skipConfirm,
 				gOpt.OptTimeout,
 				gOpt.SSHTimeout,
-				gOpt.NativeSSH,
+				gOpt.SSHType,
```

Comment:
We should use the sshtype recorded in `meta` file if users doesn't specify the `--ssh`?

Reply:
Yes, It should be.